pyaedt/modules/solutions.py

```diff
@@ -2420,7 +2465,19 @@ def _rotation_to_euler_angles(R):
 
 
 class FfdSolutionDataExporter(FfdSolutionData):
-    """Export far field solution data.
+    """Class to enable export of embedded element pattern data from HFSS.
+
+    An instance of this class is returned from the method
+    :meth:`pyaedt.Hfss.get_antenna_ffd_solution_data`. This method allows creation of
```

Comment:
```suggestion
    :meth:`pyaedt.Hfss.get_antenna_ffd_solution_data` method. This method allows creation of
```
You should put the object (method) after the tagged code entity, not before.